run_sdk_container

```diff
@@ -45,11 +45,13 @@ usage() {
     echo "      -C           - Use an entirely custom container image instead of the SDK's"
     echo "                     $sdk_container_common_registry/flatcar-sdk-[ARCH]:[SDK VERSION]"
     echo "                     Useful for CI."
+    echo "      -U             Do not update the versionfile. Instead, use the version from the versionfile as-is."
```

Comment:
Lovely, I was thinking about the same today.

Reply:
Great minds... :smile_cat: